R/pkg/R/functions.R

```diff
@@ -3803,3 +3803,89 @@ setMethod("repeat_string",
             jc <- callJStatic("org.apache.spark.sql.functions", "repeat", x@jc, numToInt(n))
             column(jc)
           })
+
+#' is_grouping
+#'
+#' Indicates whether a specified column in a GROUP BY list is aggregated or not,
+#' returns 1 for aggregated or 0 for not aggregated in the result set.
```

Comment:
As far as I remember this is actually SQL:1999 standard. And has direct relation to binary encoding used for grouping id.

Maybe `is_grouping` is not the most fortunate choice of name, but I don't have a better idea. Theoretically we can use just `grouping` and let users worry about disambiguation.